src/lib/components/multiDropzone/UploadFileCell/index.tsx

```diff
@@ -51,7 +60,8 @@ const UploadFileCell: React.FC<Props> = ({
           <div className={`p-p wmx5 ${styles['upload-display-text']}`}>
             {mapDisplayText[uploadStatus]}
           </div>
-          {isUploading && (
+
+          {isUploading && showProgressBar && (
```

Comment:
Came a little bit late to the discussion but I think it'd be ideal to also add some regression tests to this component now that we'll be working on it.
Can you create a follow up ticket to take care of testing?

Reply:
Sounds good, component doesn't have any tests so I'll create a ticket to test this and other functionalities of the component.